src/commands/repos/MergeSkeleton.ts

```diff
@@ -41,53 +42,47 @@ export class MergeSkeleton extends AbstractReposCommand {
 
     protected selectedSkeletonBranch: string;
     protected targetBranchIsTracked: boolean = false;
-    protected baseBranch: string = '';
+    protected baseBranch: string;
     protected status: IGitStatus;
     protected mergeSuccess: boolean = true;
 
     public async execute(): Promise<void> {
         const pathToRepo = path.join(process.cwd());
         const repo = new Repository(pathToRepo);
+        this.mergeSuccess = true;
```

Comment:
Aint this already true as it is set above (L47)?

Reply:
yes, this seems to be obsolete